src/gui/src/Mixer/MixerLine.cpp

```diff
@@ -270,6 +274,9 @@ float MixerLine::getVolume()
 void MixerLine::setVolume( float value )
 {
 	m_pFader->setValue( value );
+	char m_pVolumePos[7];
+	snprintf( m_pVolumePos, 6," %#.2f", value);
+	m_pFader->setToolTip( tr("Volume") + QString( m_pVolumePos ) );
```

Comment:
Could you use the Qt string library instead of the more error prone C-style char array?

```c++
ref->setToolTip(tr("Volume") + QString( " %1" ).arg( value, 0, 'f', 2 ) );
```

Reply:
Also, we should remove the tooltip of `m_pNameWidget` (in L. 162) since it probably annoy users using your new tooltips on a regular basis. In addition, it does not add much information anyway.